crates/goose-llm/src/prompts/system.md

```diff
@@ -1,6 +1,6 @@
 {{system_preamble}}
 
-The current date is {{current_date_time}}.
+The current date is {{current_date}}.
```

Comment:
does it include time still?

Reply:
nope cause the time invalidates cache so costs more $$$